tests/test_tool_registry.py

```
@@ -412,7 +455,23 @@ def test_delete_on_verify_iff_output_is_verifiable():
+                )
+            elif verifiable:
+                assert mode.mode in _DELETE_ON_VERIFY_DECLINED, (
+                    f"{mode.mode}: outputs {sorted(outs)} are verifiable against "
+                    f"{sorted(vexts)}, so delete-on-verify should be enabled — or "
+                    f"added to _DELETE_ON_VERIFY_DECLINED with the reason why not"
+                )
+
+
+def test_declined_delete_on_verify_list_is_not_stale():
+    modes = {m.mode for t in registry.all() for m in t.modes}
+    assert _DELETE_ON_VERIFY_DECLINED <= modes
```

Ruff (0.16.1)

[warning] Yoda condition detected

Rewrite as `modes >= _DELETE_ON_VERIFY_DECLINED`

(SIM300)